01-challenge-problem.py

```diff
@@ -1,17 +1,17 @@
-class Point:
+class point:
     def __init__(self, coordX, coordY):
         self.coordX = coordX
         self.coordY = coordY
 
 
 class Rectangle:
-    def __init__(self, starting_point, broad, high):
+    def __init__(self, starting_point, width, height):
         self.starting_point = starting_point
-        self.broad = broad
-        self.high = high
+        self.width = width
+        self.height = height
 
     def area(self):
```

Comment:
Naming functions should be made according to the overall functionality of the function, so a better name would be getArea().